docs/standard/toc.md

```diff
@@ -1,7 +1,7 @@
 # [Get Started with .NET](get-started.md)
 # [Tour of .NET](tour.md)
 # [.NET Architectural Components](components.md)
-# [.NET Standard Library](library.md)
```

Comment:
Should the article URL/file name change too?

I think it's kind of weird that the ".NET Standard" article has the URL https://docs.microsoft.com/en-us/dotnet/articles/standard/library.